dist/types/src/components/LoadingSpinner/index.d.ts

```diff
@@ -16,11 +16,27 @@ declare const styledSpinner: React.ComponentType<Pick<Pick<PropTypes.InferProps<
      */
     padding: PropTypes.Requireable<number>;
     classes: PropTypes.Validator<object>;
-}>, "classes"> & {
-    size?: number | null | undefined;
-    transparent?: boolean | null | undefined;
-    fixed?: boolean | null | undefined;
-    absolute?: boolean | null | undefined;
-    padding?: number | null | undefined;
-} & {}, "size" | "transparent" | "fixed" | "absolute" | "padding"> & import("@material-ui/core").StyledComponentProps<string>>;
+}>, "classes"> & Partial<Pick<PropTypes.InferProps<{
+    /**
+     * Adjust the size of the spinner
+     */
+    size: PropTypes.Requireable<number>;
+    /**
+     * I don't know what that does
+     */
+    transparent: PropTypes.Requireable<boolean>;
+    fixed: PropTypes.Requireable<boolean>;
+    absolute: PropTypes.Requireable<boolean>;
```

Comment:
fixed and absolute allow you to position the spinner, eg: on the whole page vs inside the component